app/test_main.py

```diff
@@ -1,3 +1,30 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    def test_should_be_equal_zero(self):
+        # if age less than 15
+        assert get_human_age(0, 0) == [0, 0]
+        assert get_human_age(3, 2) == [0, 0]
+        assert get_human_age(12, 4) == [0, 0]
+        assert get_human_age(14, 14) == [0, 0]
```

Comment:
The test cases here are valid, but consider using `@pytest.mark.parametrize` to avoid repetition and improve readability, as suggested in the checklist.